### DISCUSSION TITLE
Understanding the input data for clause cat component

### DISCUSSION THREAD

**Discussion by shrinidhin** [explosion/healthsea#5] [Q&A] [answered]
Hello. I am trying to use the healthsea split and blind approach for a similar use case. I am trying to train it on my own data set. While I understand how the data is to annotated, I see that in the json file for annotations there is data which has multiple entities like this:

`{"text":"Raised my <BENEFIT>\nDoesn\ut cause any <CONDITION> or <CONDITION>","meta":{"entities":"[iron level, stomach trouble, constipation]"},"_input_hash":-405931100,"_task_hash":147943101,"options":[{"id":"POSITIVE","text":"POSITIVE"},{"id":"NEGATIVE","text":"NEGATIVE"},{"id":"NEUTRAL","text":"NEUTRAL"},`

Is this how it is supposed to be tagged?If yes then how do we mention the sentiment for each entity label? I am currently only including examples like this:
```

"text":"Raised my iron level. \nDoesn\ut cause any <CONDITION> or constipation"

"text":"Raised my <BENEFIT>. \nDoesn\ut cause any stomach trouble or constipation"

"text":"Raised my iron level. \nDoesn\ut cause any stomach trouble or <CONDITION>"
```

Which is the right approach?

**Comment by thomashacker** [comment, marked as answer]
Hello,
Sorry for the late reply! Sharing our discussion on the spaCy discussion forum (https://github.com/explosion/spaCy/discussions/10194) as answer for this one 😄

**Reply by shrinidhin** [reply]
@thomashacker Yes, Thank you! Just another quick question though, I already have a tagged corpus for training my NER component. Can I use the same corpus of texts and apply the segmentation logic to it in order to label them for the sentiment analysis i.e. textcat component?Since we need data with their entities tagged, and I have already tagged a set of data for training the NER model.
